table/get_context.h

```diff
@@ -149,6 +149,8 @@ class GetContext {
 
   bool NeedTimestamp() { return timestamp_ != nullptr; }
 
+  inline size_t TimestampSize() { return ucmp_->timestamp_size(); }
```

Comment:
Can you move these two function declarations to the private section?

Reply:
I think for these two methods we can leave them as public.

The NeedTimestamp() is being used in [table_cache.cc](https://github.com/facebook/rocksdb/blob/60de713e1555ad48779c1b2ccec4d1491e64e21a/db/table_cache.cc#L480). TimestampSize() is being used in [replayGetContextLog()](https://github.com/facebook/rocksdb/blob/694e49cbb1cff88fbb84a96394a0f76b7bac9e41/table/get_context.cc#L596) and this replayGetContextLog() is designed to be a helper function outside GetContext class.

Reply:
'appendToReplayLog' can be private, right?